build.gradle

```diff
@@ -17,6 +17,14 @@ dependencies {
     testCompile group: 'junit', name: 'junit', version: '4.12'
     compile group: 'net.lingala.zip4j', name: 'zip4j', version: '1.3.2'
     compile 'com.github.crowdin:crowdin-api-client-java:1.1.7'
+
+    testCompile('org.junit.jupiter:junit-jupiter:5.5.2')
+    testCompile('org.hamcrest:hamcrest:2.2')
+    testCompile('org.mockito:mockito-core:2.1.0')
+}
+
+test {
```

Comment:
Try to add new CI Gradle task for running tests, after the build step

Reply:
I think they are already running in Azure Pipelines. 'Checks' says that the tests were run

Reply:
hmm, really :)